demos/link-prediction/homogeneous-comparison-link-prediction.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<p>Maybe this function could be renamed to something like <code>train_and_evaluate</code>  since that describes the task a little better than just "get result".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Good idea! I will rename it to <code>train_and_evaluate</code>.</p>